smart_fact_crawler/__init__.py

```diff
@@ -150,6 +150,13 @@ def sipm_voltages(url=smartfacturl + 'voltage.data'):
 
 def status(url=smartfacturl + 'status.data'):
     tc = TableCrawler(url)
+
+    value, unit = tc[28, 1].split(' ')[:2]
+    storage_newdaq = Quantity(s2f(value), unit)
+
+    value, unit = tc[29, 1].split(' ')[:2]
+    storage_daq = Quantity(s2f(value), unit)
+
```

Comment:
Probably the best we can do.
I was first afraid that the second field might not be guaranteed to be present, but the smartfact.cc actually always puts the 'B' in the second field for the unit.

smartfact.cc line 3184
``
out << col << '\t' << Tools::Scientific(freedaq) << "B\n";
``
and 3189
``
ofstream(fPath+"/status.data") << out.str();
``